### PR TITLE
Fix for copying geometry with interleaved buffers

### PR DESCRIPTION
**Pull request by benaadams**

### CODE REVIEW

File: src/core/BufferGeometry.js [hunk 1]
```diff
@@ -102,13 +102,32 @@ THREE.BufferGeometry.prototype = {
...
 			var attribute = attributes[ name ];
 
-			this.addAttribute( name, attribute.clone() );
+			if ( attribute instanceof THREE.InterleavedBufferAttribute ) {
 
+				if ( interleavedBuffers === undefined ) interleavedBuffers = {};
+
+				var buffer = interleavedBuffers[ attribute.data.uuid ];
+
+				if ( !buffer ) {
```
Comment (mrdoob) [inline src/core/BufferGeometry.js]: `buffer !== undefined`, `buffer !== null`, `buffer !== true`, ...? 😝

Comment (benaadams) [inline src/core/BufferGeometry.js]: Bad habits...

**Comment by benaadams** [comment]
Closing

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6472

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6472

**Comment by threejsworker** [comment]
The examples of this pullrequest are now built and visible in threejsworker. To view them, go to the following link: 

 http://threejsworker.com/viewpullrequest.html#6472

**Pull request closed without merging**